platform/Fetcher.ts

```diff
@@ -0,0 +1,450 @@
+type CertVerificationStatus =
```

Comment:
This is extremely messy. Do we really need everything? If yes please split this up in several files in a folder Fetcher.